### PR TITLE
Update for 1.1, primary moved to SpaceDock

### PR DESCRIPTION
**Pull request by lOmicronl**
Updates Near Future Propulsion.

Added version checking (hope I did that right?), removed some deprecated relationships.

### CODE REVIEW

**Comment by politas** [comment]
Very happy to see these turn up! Some of my favourite mods. Assuming the information on Spacedock is the same, you could dispense with the name, author and abstract.

**Comment by lOmicronl** [comment]
*shrug* It's kinda always been in there. Also, I like writing custom abstracts, so that they are shorter and more to the point. :P

File: NetKAN/NearFuturePropulsion.netkan [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
-    },
+    "spec_version"        : "v1.4",
+    "identifier"          : "NearFuturePropulsion",
+    "$kref"               : "#/ckan/spacedock/557",
+    "name"                : "Near Future Propulsion",
+    "abstract"            : "Advanced electric engines in a stockalike aesthetic.",
+    "author"              : "Nertea",
+    "license"             : "CC-BY-NC-SA-4.0",
+    "$vref"               : "#/ckan/ksp-avc",
+    "x_netkan_license_ok" : true,
```
Comment (plague006) [inline NetKAN/NearFuturePropulsion.netkan]: `x_netkan_license_ok` is redundant when we have a hardcoded `license`

**Comment by plague006** [comment]
> I like writing custom abstracts, so that they are shorter and more to the point. :P 

If you have the author's blessing then that's cool, or if you're changing the mod author's incorrect or unhelpful `abstract` then we're happy to have better information for our users. However if you're just changing `abstract` because you prefer your writing style we would prefer to have the mod author's own words.

**Comment by lOmicronl** [comment]
I'm sort of "officially" maintaining the CKAN listings forn Nertea, who has no interest in doing so himself. He's pretty much told me to go do hatever I think is best in that regard.

As for the license thing - okay, makes sense I guess. I copied the x_netkan thing from the SpaceDock guide without really understanding what it does, as it isn't explained very well there. The reason all these files have hardcoded licenses is because at some point in the past they weren't being pulled properly from Kerbalstuff, and since it worked fine after they were in there, I never bothered to remove them again.

I'll go and remove the extraneous line. Afterwards things should be good to go?

**Comment by plague006** [comment]
:+1:

**Pull request merged**